platform/machine/aws/cluster.go

```diff
@@ -84,7 +84,7 @@ func (ac *cluster) NewMachine(userdata string) (platform.Machine, error) {
 
 	conf.CopyKeys(keys)
 
-	instances, err := ac.api.CreateInstances(ac.Name(), ac.Name(), conf.String(), 1, true)
+	instances, err := ac.api.CreateInstances(ac.Name(), ac.Name(), conf.String(), 1)
```

Comment:
It seems possible a batch in a request could complete quicker than a batch of different requests depending on how placement works?

Since that's my (minor) preference, I guess it makes sense to keep it as-is and move to using the argument in the future. I'll properly review the rest now. ...